### PR TITLE
pub mod receipt

### PR DESCRIPTION
**Pull request by Thoralf-M**
# Description of change

pub mod receipt

## Type of change

- Enhancement (a non-breaking change which adds functionality)

## Change checklist

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes
- [ ] I have updated the CHANGELOG.md, if my changes are significant enough

### CODE REVIEW

File: bee-message/src/payload/milestone/option/mod.rs [hunk 1]
```diff
@@ -1,8 +1,8 @@
 // Copyright 2022 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
-mod pow;
-mod receipt;
+pub mod pow;
+pub mod receipt;
```
Comment (thibault-martinez) [inline bee-message/src/payload/milestone/option/mod.rs:5]: What do you need from these that is not already exposed ?

Comment (Thoralf-M) [reply bee-message/src/payload/milestone/option/mod.rs:5]: The dto

Comment (thibault-martinez) [reply bee-message/src/payload/milestone/option/mod.rs:5]: They are accessible through `milestone::dto::{...}`

Comment (Thoralf-M) [reply bee-message/src/payload/milestone/option/mod.rs:5]: How exactly?
```
error[E0603]: struct `ReceiptMilestoneOption` is private
   --> src/message_handler.rs:6:69
    |
6   |         payload::milestone::{dto::MilestonePayloadDto, option::dto::ReceiptMilestoneOption},
    |                                                                     ^^^^^^^^^^^^^^^^^^^^^^ private struct
    ```

Comment (Thoralf-M) [reply bee-message/src/payload/milestone/option/mod.rs:5]: How exactly? 
```
error[E0603]: struct `ReceiptMilestoneOption` is private
   --> src/message_handler.rs:6:69
    |
6   |         payload::milestone::{dto::MilestonePayloadDto, option::dto::ReceiptMilestoneOption},
    |                                                                     ^^^^^^^^^^^^^^^^^^^^^^ private struct
```
  `dto::ReceiptMilestoneOption}` also fails

Comment (thibault-martinez) [reply bee-message/src/payload/milestone/option/mod.rs:5]: But you're trying to import the base types through the dto modules

Comment (Thoralf-M) [reply bee-message/src/payload/milestone/option/mod.rs:5]: Ah right, my bad

**Pull request closed without merging**